profiles/arch/x86/package.use.mask

```diff
@@ -1,6 +1,10 @@
 # Copyright 1999-2024 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
+# Andreas Sturmlechner <asturm@gentoo.org> (2024-09-12)
+# dev-python/PyQt6 and dev-python/pyside6 aren't keyworded here
+dev-python/QtPy pyqt6 pyside6
```

Comment:
self-note: I'll try to remember to drop pyqt6 here after https://bugs.gentoo.org/935070 (been meant to have x86 for a while, even stable, but been stalled for some reason)